tests/integration/components/container-query-test.js

```diff
@@ -10,15 +10,33 @@ module('Integration | Component | container-query', function(hooks) {
   hooks.beforeEach(function(assert) {
     assert.areFeaturesCorrect = (features = {}) => {
       for (const [featureName, meetsFeature] of Object.entries(features)) {
-        if (meetsFeature) {
-          assert.dom(`[data-test-feature="${featureName}"]`).hasText('true');
-
-        } else if (meetsFeature === false) {
-          assert.dom(`[data-test-feature="${featureName}"]`).hasText('false');
-
-        } else if (!meetsFeature) {
-          assert.dom(`[data-test-feature="${featureName}"]`).hasNoText();
-
+        switch (meetsFeature) {
```

Comment:
I used a `switch` statement to be precise about what values we can pass. I think this is preferable to the previous approach, which makes an indirect inference (i.e. looks at whether the value is truthy or falsey).

We could create a `default` case to throw an error if we passed an incorrect value, but I don't think that's necessary at this moment.